bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
```

Comment:
this needs to go

Reply:
Just to be clear:
I kept "import model loop_sm-ckm_no_b_mass" line 
and removed other commented lines.